.github/ISSUE_TEMPLATE/06_localization_data_issue.yml

```diff
@@ -0,0 +1,104 @@
+          * For date/time elements, consider the advice and observations in Jon Skeet's post [_Storing UTC is not a silver bullet_](https://codeblog.jonskeet.uk/2019/03/27/storing-utc-is-not-a-silver-bullet/).
+        * You **MUST NOT** [consider culture data to be stable](https://learn.microsoft.com/en-us/archive/blogs/shawnste/culture-data-shouldnt-be-considered-stable-except-for-invariant).  This applies to both format strings and other culture data like timezone rules.
+        * To make dependencies more explicit and testing potentially easier and more parallelizable, you _SHOULD CONSIDER_ explicitly passing culture-relevant settings, such as `CultureInfo` parameters, to methods that perform formatting or `TimeZoneInfo` to methods that work with dates and times.  You **MUST** use [`TimeProvider`](https://learn.microsoft.com/en-us/dotnet/api/system.timeprovider?view=net-8.0) or a similar type when retrieving the time.
+        * For most tests, you _SHOULD NOT_ explicitly validate the exact output of a given formatting operation or the exact offset of a timezone.  Formatting and timezone data can change at any time, and may differ between two otherwise identical instances of an operating system (and potentially different processes on the same machine).  Relying on an exact value makes tests brittle.
+          * Generally, validating that some output was received will be sufficient (eg, non-empty strings when formatting).
+          * For some data elements and formats, validating that the data parses to the input value may be used instead (roundtripping).  Care needs to be taken for cases where fields are dropped (eg, year for some date-related fields) or the value truncated or rounded (such as for floating-point output).
+          * If you have explicit requirements to validate all localized format output, you _SHOULD CONSIDER_ creating and using a custom culture.  This can be done by creating a custom `CultureInfo` object, and can be entirely stable.  There are potential additional benefits to this, such as enabling [pseudolocalization](https://learn.microsoft.com/en-us/globalization/methodology/pseudolocalization) via resource files.
+          * If you have explicit requirements to validate the results of all date/time operations, you _SHOULD CONSIDER_ creating and using a custom `TimeZoneInfo` instance.  There are potential additional benefits to this, such as enabling stable testing of certain edge cases (eg, changes to DST rules).
```

Comment:
Will be good to emphasize it is not recommended using custom time zones in the production.